### ISSUE TITLE
float64 setting in keras.json makes Conv2D layer in tf backend error

### ISSUE THREAD

**Issue by ShieLian** [fchollet/keras#7176]
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

dtype error will occure when add a Conv2D layer, with floatx='float64' in keras.json
Code causes error:

from keras.models import Sequential
from keras.layers import Conv2D
model=Sequential()
model.add(Conv2D(filters=64, kernel_size=(9,9), padding='valid',input_shape=(33,33,1),activation='relu'))

**Comment by Dref360** [comment]
According to this page : 
https://www.tensorflow.org/api_docs/python/tf/nn/conv2d
conv2d only supports float32. So it is the expected behaviour

**Comment by ozabluda** [comment]
There is PR for this now: https://github.com/tensorflow/tensorflow/pull/12943

**Comment by aakanksharathore** [comment]
I am also having same issue, did anyone find a solution?
